SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -1118,3 +1139,60 @@ void ASpatialFunctionalTest::ClearAllTakenSnapshots()
 	bWasLoadedFromTakenSnapshot = false;
 	TakenSnapshots.Empty();
 }
+
+void ASpatialFunctionalTest::GenerateMap()
+{
+	GeneratedTestMap->GenerateMap();
```

Comment:
Here (and below) GenerateTestMap could be nullptr? Is this a problem?

Reply:
Typically I would whenever I use a pointer, however it might be overkill here - will close for now.